lib/hyku_addons/engine.rb

```diff
@@ -495,6 +495,10 @@ def self.dynamically_include_mixins
       Image.include HykuAddons::ImageOverrides
       GenericWork.include ::Hyrax::BasicMetadata
       Hyrax::WorkIndexer.include HykuAddons::WorkIndexerBehavior
+
+      # HykuAddons::DOIFormBehavior must be prepended before WorkForm overrides
+      Hyrax::DOI::DOIFormBehavior.prepend HykuAddons::DOIFormBehavior
```

Comment:
Thanks for your helpful feedback @mankind this morning.  I tried `prepend`, which I hadn't before, as that was the correct way to override this class.  Still didn't work, but eventually I realised this needed to be prepended before the WorkForm overrides.